### PR TITLE
kernel: Update kernel  and image versions

### PR DESCRIPTION
**Pull request by jcvenegas**
Update kernel version to 4.9.31-68 and remove cve fixed
in this version.

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by gorozco1** [comment]
Need to update also linux-containers.spec-template and wd/debian/patches/series too 👎

**Comment by jcvenegas** [comment]
@gorozco1 updated templates.

**Comment by chavafg** [comment]
qa-failed

**Comment by gorozco1** [comment]
lgtm

**Comment by miguelinux** [comment]
+1

**Comment by jcvenegas** [comment]
moving to `do-not-merge` until obs packages are generated correctly

File: data/obs-packaging/linux-container/linux-container.spec-template [hunk 1]
```diff
@@ -33,9 +33,6 @@ BuildRequires:  bison
 %define debug_package %{nil}
 %define __strip /bin/true
 
-# Serie    00XX: mainline, CVE, bugfixes patches
```
Comment (gorozco1) [inline data/obs-packaging/linux-container/linux-container.spec-template]: missing removing the part where you apply this patch %p1

**Comment by gorozco1** [comment]
@jcvenegas  Please add also the changes generated in the changelogs for the clear-containers-image and linux-container when updating for OBS

**Comment by gorozco1** [comment]
👍

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**